src/PendingCalls/TestCall.php

```diff
@@ -642,9 +646,25 @@ public function __get(string $name): self
      */
     public function __call(string $name, array $arguments): self
     {
+        if (self::hasExtend($name)) {
+            $extend = self::$extends[$name]->bindTo($this, TestCall::class);
+
+            if ($extend != false) {
+                return $extend(...$arguments);
+            }
+        }
+
         return $this->addChain(Backtrace::file(), Backtrace::line(), $name, $arguments);
     }
 
+    public function extend(string $name, Closure $extend): void
+    {
+        $this->traitExtend($name, $extend);
+
+        $this->description = "extend: $name";
+        $this->testCaseMethod->closure = fn (): null => null;
```

Comment:
This is required because `extend` is being called non-statically. With this combination no errors are thrown and no output is included from a `test()->extend(...)` call.